File: Lab_1/1.1_lab.cpp
#include <iostream>
#include <string>
using namespace std;

string highestScoringWord(const string &str);

int main() {
	
cout << highestScoringWord("London is a capital of great Britain") << endl;
	
return 0;
}

string highestScoringWord(const string &str) {
	int max = 0, score = 0, len = str.length();
	string max_word = "", word = "";
	for(int i = 0; i < len + 1; i++) {
		if(str[i] != ' ' && i != len) {   
			word += str[i];
			score += tolower(str[i]) - 'a' + 1;
		}
		else {
			if(score > max) {
				max = score;
				max_word = word;
			}
		score = 0;
		word = "";
		}	     
	}
return max_word;
}

File: Lab_1/1.1_lab.py
alph = 'abcdefghijklmnopqrstuvwxyz'

def high(str):
	arr = str.split(' ')
	list = []
	max_word = ''
	value = 0 
	for word in arr:
		for ch in word:
			value += alph.index(ch) + 1	
		list.append(value)	
		value = 0	
	max_word = arr[list.index(max(list))]
	return max_word

print(high('London is a capital of great Britain'))
